client/src/components/clientHomeEditor/ClientHomeEditor.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * This file exports a page for client to edit content in their homepage.
+ */
+
+import React, { useState } from 'react';
+import { Form, Container, Button } from 'react-bootstrap';
+import './ClientHomeEditor.css';
+
+const ClientHomeEditor = () => {
+  // content/values
+  const [homepageValues, setValues] = useState({
+    aboutUs: [''],
```

Comment:
Just wondering why the about us string is inside an array?

Reply:
Because I want to store different paragraph individually to better customize the style on client site.